.github/workflows/_rust.yml

```diff
@@ -83,11 +83,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - runs-on: ubuntu-20.04
+          - runs-on: ubuntu-20.04-firezone
           # Broken on 22.04 <https://github.com/firezone/firezone/issues/3699>
-          - runs-on: ubuntu-22.04
-          - runs-on: windows-2019
-          - runs-on: windows-2022
+          - runs-on: ubuntu-22.04-firezone
+          - runs-on: windows-2019-firezone
+          - runs-on: windows-2022-firezone
```

Comment:
At least for these, there seems to be virtually no difference to the regular runners. Doesn't appear to be a CPU bottle neck. I noticed that downloads are pretty slow so maybe there is still some Windows Defender shit going on that throttles IO in general?